### DISCUSSION TITLE
Trying to uninstall v3.2 64 bit, but can't find it anywhere?

### DISCUSSION THREAD

**Discussion by Cully** [sumatrapdfreader/sumatrapdf#3546] [Q&A] [answered]
Hiya,

I installed v3.2 x64 ages ago and a few months ago I installed v3.4.6 x64.

v3.4.6 works perfectly, but for some reason, when I right click an ebook file, I get 2 options. One is 3.4.6 and one is 3.2. It doesn't tell me which is which. If I uninstall sumatra from add/remove programs, I only have v3.2 on my context menu. 

I want to keep v3.4.6 but remove the v3.2 version. I cannot find the EXE file anywhere and I cannot see it installed anywhere. Do you know how I can delete v3.2 from the context menu, or ideally, uninstall v3.2 completely? I am on Windows 11. 

Thanks

**Comment by GitHubRulesOK** [comment, marked as answer]
hmm interesting there was a shift in storage between the two versions which  seems to be your issue, so not wishing to cause more problems, 
I would recommend a double install. backup your settings to say the desktop in case you forgot to keep any favourites / customisations etc. 
Ensure 3.4.6 is not active and only the bothersome 3.2

1) determine which variant of 3.2 is current is it 32bit or 64bit (Help About should either say 64bit or not :-)
2) ensure you have/download the same installer (links are shown here for 3.2 https://www.sumatrapdfreader.org/download-prev)
3) try to go through a similar install (did you use extra options previously) It should overwrite the current working copy
4) ***now you should be able to run a clean uninstall 3.2*** (some data may be left, but new 3.4.6 should potentially use some of those settings ??

**5) check there is no longer any working SumatraPDF** (if there is then do not install again)
Icons should be Red for Edge and Edge should be the default for pdf opening, 
probably best at this point to right click a  pdf and ensure there is no SumatraPDF icon.

6) Install your desired version

**Reply by Cully** [reply]
Thanks for this. It worked perfectly. 

I uninstalled all, reinstalled v32 x64 and then uninstalled. There was nothing on my system. Then I reinstalled v3.4.6 and it is the only option now.

I appreciate the help!
